spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -101,6 +101,17 @@
           end
         end
 
+        context 'colors passed as comma-separated values in the query parameters' do
+          it 'handles them fine' do
+            get "/api/v0/highlights?source_type=openstax_page&scope_id=#{scope_1_id}&" \
+                "colors=yellow,pink"
```

Comment:
does this need a test that tests that this test is not a false positive?  (e.g., testing for just yellow returns a subset of the yellow, pink?

Reply:
This test already excludes the new `"green"` highlight